LICENSE.txt

```diff
@@ -17,7 +17,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 Mautic is released under the GPL v3.
 
-Mautic is a registered trademark of David Hurley.
+Mautic is a registered trademark held by Open Source Collective.
```

Comment:
```suggestion
Mautic is a trademark of the Mautic project of the Open Source Collective.
```
This is how we'd agreed with Open Source Collective that we'd write it on our website so I think it should be fine here. If needed we can link out to https://contribute.mautic.org/policies/trademark?